src/Command/PullRequest/PullRequestPatOnTheBackCommand.php

```diff
@@ -48,11 +81,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $adapter = $this->getAdapter();
         $pr = $adapter->getPullRequest($prNumber);
+        $config = $this->getConfig();
+
+        if (!$pats = $config->get('pats')) {
+            $pats = Pats::getDefaultPats();
+        }
+
+        $pat = $input->getOption('pat');
+
+        if ($input->getOption('random')) {
+            $pat = Pats::getRandomPatName(array_keys($pats));
+        } elseif (!$pat) {
+            $pat = $this->choosePat($pats);
+        } elseif (!array_key_exists($pat, $pats)) {
+            throw new UserException(sprintf('Pat "%s" doesn\'t exist', $pat));
```

Comment:
I agree @sstok, but it would require a way to hydrate the `Pats` object with external pats; in this case, which come from `.gush.yml` file. I'll try to find a better approach.